src/cljs/main/broadfcui/common/method/messages.cljs

```diff
@@ -0,0 +1,15 @@
+(ns broadfcui.common.method.messages
+  (:require
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.style :as style]
+   [broadfcui.utils :as utils]
+   ))
+
+
+(defn get-methods-repo-group-alert []
+  [:div {:style {:paddingBottom "1em" :fontSize "100%"}}
```

Comment:
If you make this a `p` you probably won't need the padding; also, avoid `em` for `rem`